Makefile

```diff
@@ -1,30 +1,72 @@
+.PHONY: all cover fast-cover generate-proto test test-all test-all-race gometalinter
 
-.PHONY: all cover fast-cover generate-proto test test-all
-
+PILOSA_VERSION ?= master
+PILOSA_VERSION_ID := pilosa-$(PILOSA_VERSION)-$(GOOS)-amd64
+PILOSA_DOWNLOAD_URL ?= https://s3.amazonaws.com/build.pilosa.com/$(PILOSA_VERSION_ID).tar.gz
 VERSION := $(shell git describe --tags 2> /dev/null || echo unknown)
+GOOS ?= linux
+PILOSA_BIND ?= http://:10101
 
 all: test
 
-cover:
-	go test ./... -cover -tags="integration fullcoverage" $(TESTFLAGS)
+cover: vendor
+	PILOSA_BIND=$(PILOSA_BIND) go test ./... -cover $(TESTFLAGS) -covermode=count -coverprofile=build/coverage.out
 
 fast-cover:
-	go test ./... -cover -tags="integration" $(TESTFLAGS)
+	PILOSA_BIND=$(PILOSA_BIND) go test ./... -cover -tags="nointegration" $(TESTFLAGS)
```

Comment:
I think we should remove `nointegration` tag, so integration tests run for this command.